components/toast/toast.css

```diff
@@ -0,0 +1,34 @@
+/* || TOAST */
+.toast {
+  display: inline-flex;
+  background-color: var(--link-color);
+  color: var(--badge-count-text);
+  border-radius: 0.3rem;
+  padding: 0.3rem 1.5rem;
+}
+
+/* || STACKED TOAST */
+.toast-stacked {
+  flex-direction: column;
+}
+
+.toast-text,
+.toast-actions {
+  margin: 0.5rem 0;
```

Comment:
suggestion - you can use px for values less than 1 rem

Reply:
Just being consistent throughout the project. Hope its fine.